### DISCUSSION TITLE
any timeframe of next release?

### DISCUSSION THREAD

**Discussion by apws** [SpenceKonde/megaTinyCore#602] [Future Development Discussion]
hi, may I ask how its going with new and fixed things, when we can expect next release? thanks
(manual installation requirements really ARE brutal :-)

**Comment by SpenceKonde** [comment]
Today, Of course f you asked me the same thing any time in the past 4 days
you'd have gotten te same answer... "today"

On Mon, Jan 10, 2022 at 6:00 AM apws ***@***.***> wrote:

> hi, may I ask how its going with new and fixed things, when we can expect
> next release? thanks
> (manual installation requirements really ARE brutal :-)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SpenceKonde/megaTinyCore/discussions/602>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABTXEW64BVR7VRLT7NKJLS3UVK34VANCNFSM5LTLF2JA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore <https://github.com/SpenceKonde/ATTinyCore>: Arduino support for
all pre-2016 tinyAVR with >2k flash!
megaTinyCore <https://github.com/SpenceKonde/megaTinyCore>: Arduino support
for all post-2016 tinyAVR parts!
DxCore <https://github.com/SpenceKonde/DxCore>: Arduino support for the AVR
Dx-series parts, the latest and greatest from Microchip!
Contact: ***@***.***

**Reply by apws** [reply]
))

**Reply by apws** [reply]
I dont want to distract ... let it settle well :-)
